pom.xml

```diff
@@ -13,8 +13,8 @@
 
 		<version.jetty>9.4.0.v20161208</version.jetty>
 		<version.lyo>2.4.0-SNAPSHOT</version.lyo>
-		<version.shaclex>0.0.63</version.shaclex>
-		<jena.version>3.0.1</jena.version>
+		<version.shaclex>0.0.76</version.shaclex>
+		<jena.version>3.6.0</jena.version>
```

Comment:
Are we planning to move from Jena 3.0.1 to 3.6.0 in all Lyo Projects? Was there a need to upgrade Jena Version here?

Reply:
There is no way to upgrade Jena in one project and not another. Then lyo-store 2.4.0 would be incompat with lyo-validation 2.4.0...